src/esp/nav/PathFinder.h

```diff
@@ -15,13 +15,27 @@ class dtNavMeshQuery;
 class dtQueryFilter;
 class dtQueryPathState;
 
+void dtFreeNavMesh(dtNavMesh* navmesh);
+void dtFreeNavMeshQuery(dtNavMeshQuery* navmeshQuery);
+
 namespace esp {
 // forward declaration
 namespace assets {
 struct MeshData;
 }
 namespace nav {
 
+struct NavMeshDeleter {
```

Comment:
These can be put in the class definition or in an additional `impl` namespace as to not populate the `nav` namespace.